Content/Bosses/GlassMiniboss/NPCs.Glassweaver.cs

```diff
@@ -329,11 +334,13 @@ public override void AI()
 
 		public override void SendExtraAI(BinaryWriter writer)
 		{
+			writer.WritePackedVector2(arenaPos);
```

Comment:
Questioning if this needs to be synced -- in theory this could be made a readonly property entirely based on the vitric desert's position? May need a bit of reverse engineering to look through and properly calculate but I dont think this should ever change world to world.

Reply:
Main reason to sync this is cause the first few frames of the boss fight aren't guaranteed to be ran on all clients from what I saw so without this it would intermittently de spawn. another solution is like you say with a readonly property that gets calculated on the fly